### DISCUSSION TITLE
Can the vector obtain CPU and memory information

### DISCUSSION THREAD

**Discussion by xxsdnol** [vectordotdev/vector#18450] [Q&A] [answered]
Is there a way to obtain information about the server's CPU, memory, and traffic usage when using Vector? I don't seem to see any way to obtain it in the official documents

**Comment by jszwedko** [comment, marked as answer]
Hi @XSWClevo ! Not yet. This is being tracked by https://github.com/vectordotdev/vector/issues/6155

**Reply by xxsdnol** [reply]
Thank you for your answer. Wishing you a happy life
